doc/source/kubecluster2.rst

```diff
@@ -0,0 +1,106 @@
+
+Installing the Operator
+-----------------------
+
+.. currentmodule:: dask_kubernetes
+
+First you must install the Dask Operator and have access to a Kubernetes Cluster. To install the the operator, first you'll need to deploy the custom resources
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/customresources/daskcluster.yaml
+   kubectl apply -f dask_kubernetes/operator/customresources/daskworkergroup.yaml
+
+Finally, you'll need to create the operator
+
+.. code-block:: bash
+   kubectl apply -f dask_kubernetes/operator/deployment/manifest.yaml
+   
+Creating a Dask cluster via `kubectl`
+-------------------------------------
+
+To create a cluster in the default namespace, run the following
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/tests/resources/simplecluster.yaml
+
+You can create another yaml file following the `simplecluster.yaml` file to change
+the default configuration (`namespace`, `replicas`, etc.) of your cluster.
+
+You can scale the cluster
+
+.. code-block:: bash
+
+   kubectl scale --replicas=5 daskworkergroup simple-cluster-default-worker-group
+
+Or add an additional worker group
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/tests/resources/simpleworkergroup.yaml
+
+Finally delete the cluster by running
+
+.. code-block:: bash
+
+   kubectl delete daskcluster simple-cluster
+
+Creating a Dask cluster via the cluster manager `KubeCluster2`
+--------------------------------------------------------------
+
+To create a cluster in the default namespace, run the following
+
+.. code-block:: python
+
+   cluster = KubeCluster2(name = 'foo')
+
+You can change the default configuration of the cluster by passing additional args
+to the python class (`namespace`, `n_workers`, etc.) of your cluster.
+
+You can scale the cluster
+
+.. code-block:: python
+
+   # Scale up the cluster
+   cluster.scale(5)
+
+   # Scale down the cluster 
+   cluster.scale(1)
+
+Or add an additional worker group
+
+.. code-block:: python
+
+   cluster.add_worker_group("additional")
+
+   # Scale the new worker group
+   cluster.scale(5, "additional")
+
+You can connect to the client
+
+.. code-block:: python
+
+    # Example usage
+    from dask.distributed import Client
+    import dask.array as da
+
+    # Connect Dask to the cluster
+    client = Client(cluster)
+
+    # Create a large array and calculate the mean
+    array = da.ones((1000, 1000, 1000))
+    print(array.mean().compute())  # Should print 1.0
+
+Finally delete the cluster by running
+
+.. code-block:: python
+
+   cluster.close()
+
```

Comment:
Let's add an "API" section here with an `autosummary` and `autoclass` like we do on the Helm page.